Clarify what functionality of time_bucket_ng() is supported by CAGGs (#627)
timescale/docs: api/time_bucket_ng.md

For more information, see the [continuous aggregates documentation][caggs].

> Note that although `time_bucket_ng()` supports months and timezones, it doesn't
> necessarily mean that monthly buckets or buckets with timezones are supported
> by continuous aggregates. The following table shows what functionality of
> `time_bucket_ng()` is currently supported on the continuous aggregates side:

Reviewer: ```suggestion
Note that although `time_bucket_ng()` supports months and timezones, currently
continuous aggregates do not support months and timezones when using `time_bucket_ng()`.

The following table shows what functionality of
`time_bucket_ng()` is currently supported on the continuous aggregates side:
```

Reviewer 2: ```suggestion
While `time_bucket_ng()` supports months and timezones, continuous aggregates cannot always be used with monthly buckets or buckets with timezones. 

This table shows which `time_bucket_ng()` functions can be used in a continuous aggregate:
```

> Functionality | Supported
> --------------|----------
> Buckets by seconds, minutes, hours, days and weeks | YES, since 2.4.0
> Buckets by months and years | NO (expected in 2.6.0)
> Specifying custom origin | NO
> Timezones support | NO

Reviewer 2: ```suggestion
|Function|Available in continuous aggregate|TimescaleDB version|
|-|-|-|
|Buckets by seconds, minutes, hours, days and weeks|✅|2.4.0 and later|
|Buckets by months and years|❌|Expected in 2.6.0 or later|
|Specify custom origin|❌|To be determined|
|Timezones support|❌|To be determined|
```
